mitosheet/mitosheet/transpiler/transpile_utils.py

```diff
@@ -225,6 +256,9 @@ def convert_script_to_function(
     final_code += imports
     final_code.append("")
 
+    # Get the final function param
+    function_params = get_final_function_params_from_code_options(steps_manager, function_params)
```

Comment:
It's hard to tell from looking at this what the function does, especially given the return is called `function_params` but it's also one of the params.